ci/actions-templates/linux-builds-template.yaml

```diff
@@ -131,10 +131,6 @@ jobs:
           echo "DOCKER=$DOCKER" >> $GITHUB_ENV
       - name: Fetch the docker
         run: bash ci/fetch-rust-docker.bash "${TARGET}"
-      - name: Install Protoc
-        uses: arduino/setup-protoc@v1
-        with:
-          version: '3.x'
```

Comment:
```
  with:
    repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Seems like the canonical approach to avoid rate limiting, from the setup-protoc repo.

Reply:
Added this in another commit to `all-features.yaml`.